File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
            int count1 = 0;
            int count2 = 0;
            int count3 = 0;
            
            for(int i=0; i<nums.length; i++){
                    if(nums[i] == 0){
                            count1++;
                    }else if(nums[i] == 1){
                            count2++;
                    }else{
                            count3++;
                    }
            }
            
            for(int i=0; i<count1; i++){
                    nums[i] = 0;
            }
            
            for(int i=count1; i<count2+count1; i++){
                    nums[i] = 1;
            }
            
            for(int i=count2+count1; i<count3+count2+count1; i++){
                    nums[i] = 2;
            }
    }
}

File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int count=0;
        int count1=0;
        int count2=0;
        
        for(int i=0;i<nums.size();i++){
            if(nums[i]==0){
                count++;
            }
            if(nums[i]==1){
                count1++;
            }
            if(nums[i]==2){
                count2++;
            }
        }
          for(int i =0; i< count; i++){
            nums[i] =0;
        }
        for(int i =count; i< (count1 + count); i++){
            nums[i] =1;
        }
        for(int i =(count1 + count); i< nums.size(); i++){
            nums[i] =2;
        }
    }
};